consensus/state.go

```diff
@@ -1560,6 +1542,12 @@ func (cs *State) enterPrecommit(height int64, round int32) {
 		return
 	}
 
+	if ready, waitTime := cs.isReadyToPrecommit(); !ready {
+		logger.Info("rescheduling precommit", "delay(s)", waitTime.Seconds())
```

Comment:
shouldn't it be ms? seconds isn't that helpful if the entire voting process is a couple of them